pkg/api/openshiftcluster.go

```diff
@@ -761,3 +767,38 @@ type HiveProfile struct {
 	// ClusterDeployments.
 	CreatedByHive bool `json:"createdByHive,omitempty"`
 }
+
+type PlatformWorkloadIdentityProfile struct {
+	MissingFields
+
+	PlatformWorkloadIdentities []PlatformWorkloadIdentity `json:"platformWorkloadIdentities,omitempty"`
+}
+
+// PlatformWorkloadIdentity stores information representing a single workload identity.
+type PlatformWorkloadIdentity struct {
+	MissingFields
+
+	Name       string `json:"name,omitempty"`
```

Comment:
This was originally `Name`, but we later changed it to `OperatorName` in the design doc to make the purpose of the field more clear.